src/babylon.engine.ts

```diff
@@ -2344,10 +2349,6 @@
             }
             gl.framebufferTexture2D(gl.FRAMEBUFFER, gl.COLOR_ATTACHMENT0, gl.TEXTURE_2D, texture, 0);
 
-            if (generateMipMaps) {

```

Comment:
This may be required to get a complete texture. Because this could be bind as a texture before a render is done